composer.json

```diff
@@ -37,6 +37,7 @@
         "symfony/cache": "self.version",
         "symfony/config": "self.version",
         "symfony/console": "self.version",
+        "symfony/contracts": "self.version",
```

Comment:
Should be `1.0.0` instead of `self.version`, right?

Reply:
good catch, fixed